EmailNotification/handler.ts

```diff
@@ -126,12 +139,30 @@ export const getEmailNotificationHandler = (
           const emailNotification =
             errorOrEmailNotification.right.channels.EMAIL;
 
-          const documentHtml = await generateDocumentHtml(
-            content.subject,
-            content.markdown,
-            senderMetadata
+          const useTemplate = pipe(
+            errorOrActiveMessage.right.fiscalCode,
+            getIsUserEligibleForNewFeature(
+              cf => BETA_USERS.includes(cf),
+              () => false, // NO canary implemented yet
+              FF_TEMPLATE_EMAIL
+            )
           );
 
+          const documentHtml = await (useTemplate
+            ? pipe(
+                { content, senderMetadata },
+                messageToHtml(),
+                TE.mapLeft(err => {
+                  throw err;
+                }),
+                TE.toUnion
+              )()
+            : generateDocumentHtml(
+                content.subject,
+                content.markdown,
+                senderMetadata
+              ));
+
```

Comment:
why not collapsing all in the same pipe?

Reply:
fixed, ty